packages/sdk-btc/source/client.service.ts

```diff
@@ -51,7 +51,7 @@ export class ClientService extends Services.AbstractClientService {
 			query,
 		);
 
-		return response.json();
+		return response.json().data;
```

Comment:
@faustbrian Not sure if you prefer all elements wrapped in data element (then we need to merge this one) or not (then we need to change btc-indexer endpoint).

Reply:
Always wrapped in `data` because otherwise you introduce inconsistencies between endpoints that use pagination and the ones which don't.